subsys/fb/Kconfig

```diff
@@ -21,6 +21,14 @@ config CHARACTER_FRAMEBUFFER_USE_DEFAULT_FONTS
 	help
 	  Use default fonts.
 
+config CHARACTER_FRAMEBUFFER_SHELL
+	bool "Character Framebuffer shell"
+	select SERIAL
```

Comment:
No need to depend on serial, the correct backend needs to be configured in the shell Kconfig